tests/schemas/invalid/test_invalid_relationship_list_doesnt_exist.json

```diff
@@ -305,8 +471,8 @@
                         }
                       },
                       {
-                        "value": "Legally Registered Civil Partner",
-                        "label": "Legally Registered Civil Partner",
+                        "label": "Legally registered civil partner",
+                        "value": "Legally registered civil partner",
```

Comment:
Can we reduce the schema by only having 2 options, having all these are not necessary for the test.